bin/compile

```diff
@@ -29,6 +29,8 @@ for BUILDPACK in $(cat $1/.buildpacks); do
     if [ "$branch" != "" ]; then
       git checkout $branch >/dev/null 2>&1
     fi
+    # change back to original directory for the sake of the configured buildpacks
+    cd -
```

Comment:
Maybe consider `pushd` and `popd` in case someone sneaks another `cd` in between in the future? Fine as is in the short term if you'd rather do that if/when we get there. :)

Reply:
I don't anticipate very many changes to this in the future since the average Heroku user could use multiple buildpacks in config through the CLI for their apps. This is just for our internal apps.